sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationLiveTests.cs

```diff
@@ -41,132 +41,142 @@ public DocumentModelAdministrationLiveTests(bool isAsync, DocumentAnalysisClient
+                // be ignored when running in Live mode.
                 Assert.Ignore("https://github.com/Azure/azure-sdk-for-net/issues/27042");
             }
 
             var client = CreateDocumentModelAdministrationClient();
-            var trainingFilesUri = new Uri(TestEnvironment.BlobContainerSasUrl);
             var modelId = Recording.GenerateId();
+            var startTime = Recording.UtcNow;
+            var trainingFilesUri = new Uri(TestEnvironment.BlobContainerSasUrl);
+            var prefix = "subfolder";
+            var options = new BuildDocumentModelOptions()
+            {
+                Description = $"This model was generated by a .NET test: {nameof(BuildDocumentModel)}",
+                Tags = { { "tag1", "value1" }, { "tag2", "value2" } }
+            };
 
-            BuildDocumentModelOperation operation = await client.BuildDocumentModelAsync(WaitUntil.Started, trainingFilesUri, DocumentBuildMode.Neural, modelId);
+            BuildDocumentModelOperation operation = null;
 
-            await operation.WaitForCompletionAsync(TimeSpan.FromMinutes(1));
+            try
+            {
+                operation = await client.BuildDocumentModelAsync(WaitUntil.Started, trainingFilesUri, buildMode, modelId, prefix, options);
+
+                await (buildMode == DocumentBuildMode.Neural
+                    ? operation.WaitForCompletionAsync(TimeSpan.FromMinutes(1))
+                    : operation.WaitForCompletionAsync());
+            }
+            finally
+            {
+                if (operation != null && operation.HasValue)
+                {
+                    await client.DeleteDocumentModelAsync(modelId);
+                }
+            }
 
             Assert.IsTrue(operation.HasValue);
 
             DocumentModelDetails model = operation.Value;
 
-            ValidateDocumentModelDetails(model);
-
-            Assert.AreEqual(1, model.DocumentTypes.Count);
-            Assert.IsTrue(model.DocumentTypes.ContainsKey(modelId));
-
-            DocumentTypeDetails documentType = model.DocumentTypes[modelId];
+            Assert.AreEqual(modelId, model.ModelId);
+            Assert.AreEqual(options.Description, model.Description);
+            Assert.AreEqual(ServiceVersionString, model.ApiVersion);
+            Assert.Greater(model.CreatedOn, startTime);
 
-            Assert.AreEqual(DocumentBuildMode.Neural, documentType.BuildMode);
-        }
+            if (_serviceVersion >= DocumentAnalysisClientOptions.ServiceVersion.V2023_02_28_Preview)
+            {
+                Assert.Greater(model.ExpiresOn, model.CreatedOn);
+            }
+            else
+            {
+                // We have changed the following validation because of a service bug. This needs to be updated once the bug is fixed.
+                // More information: https://github.com/Azure/azure-sdk-for-net/issues/35809
 
-        [RecordedTest]
-        public async Task BuildDocumentModelSucceedsWithValidPrefix()
```

Comment:
Test merged into `BuildDocumentModel`, which already uses a prefix to specify training files.